utils.js

```diff
@@ -0,0 +1,224 @@
+const fs = require('fs'), path = require('path');
+
+const trim = (x, characters=" \t\w") => {
+  var start = 0;
+  while (characters.indexOf(x[start]) >= 0) {
+    start += 1;
+  }
+  var end = x.length - 1;
+  while (characters.indexOf(x[end]) >= 0) {
+    end -= 1;
+  }
+  return x.substr(start, end - start + 1);
+}
+
+const cleanExternalId = (rawExternalId, dbName) => {
+  // clean rawExternalId
+  rawExternalId = trim(rawExternalId.trim(), '"');
+  if (dbName == 'MA'){
+    rawExternalId = rawExternalId.replace(/^MA-/, '');
```

Comment:
> Awesome stuff! I left some minor comments regarding improvements that are not affecting functionality. Also, I noticed in the output
> 
> ```
> Warning: cannot find external ID file subsystems.tsv in path 
> ```
> 
> but I don't remember ever being subsystem annotation that should be parsed.

@mihai-sysbio , it was included in the code in the main branch https://github.com/MetabolicAtlas/neo4j-data-generation/blob/main/index.js#L300. I kept it because I was not sure if the tsv file for subsystems were missing just because the data is incomplete. I think it can be removed if we are sure that tsv file for subsystems will not be included now and in the future.

Reply:
I don't remember anything about a `subsystems.tsv` being planned. If the only change would be to remove the word `subsystem` from that array, I feel it would be worth doing this now so that we don't get any warnings when building.

Reply:
Addressed in the commit [8a8fe54](https://github.com/MetabolicAtlas/neo4j-data-generation/pull/16/commits/8a8fe548cd44818ceb0570a88c2709549538522a)